lib/matplotlib/contour.py

```diff
@@ -124,6 +124,11 @@ def clabel(self, levels=None, *,
             of texts during the drawing time, therefore this can be used if
             aspect of the axes changes.
 
+        zorder : int, optional
+            zorder of the contour labels.
+
+            If not specified, the default zorder of `.Text` class is used.
```

Comment:
I think `zorder` is a scalar (i.e. float) or None.

I'm not clear what happens if zorder is None.  Should it not default to the zorder of the contours?

Reply:
will change the docstring to float or None.

If zorder is None, the default zorder (= 3) of text objects is used. At present, there is no logic in the code to deduce the zorder of contour lines and set the zorder of clables accordingly. Contours are just line collections hence default to zorder of 2 and labels are instances of text class hence default to 3. Even if the zorder of contours is explicitly specified, it will be difficult to determine if the results of multiple calls to ax.contour() are overlapping.

Reply:
done.

Reply:
Do I need to mention the change in whats new? Not sure if this qualifies as backward-incompatible change.